src/main/scala/com/lynbrookrobotics/sbt/Tasks.scala

```diff
@@ -218,6 +220,23 @@ object Tasks {
     }
   }
 
+  lazy val robotConsole: Def.Initialize[Task[Unit]] = Def.task {
+
+    val SIZE = 1024
+    val PORT = 6666
+    val logger = streams.value.log
+
+    val socket = new DatagramSocket(PORT)
+    while (true) {
+      val buffer = new Array[Byte](SIZE)
+      val packet = new DatagramPacket(buffer, buffer.length)
+      socket.receive(packet)
```

Comment:
According to the docs:
"When this method returns, the DatagramPacket's buffer is filled with the data received. ... This method blocks until a datagram is received."

It seems like I would have to print 1 kilobyte of messages before any data is displayed. Could this cause issues if my prints are very short?

Reply:
+1, should have a smaller buffer probably?

Reply:
receiving datagram != filling buffer
buffer should frankly be bigger to accommodate maximum udp packet size